src/amo/components/LandingPage/index.js

```diff
@@ -227,15 +209,12 @@ export class LandingPageBase extends React.Component {
         {errorHandler.renderErrorIfPresent()}
 
         <div className="LandingPage-header">
```

Comment:
Looks like this can just be a `<Card>` component and it'll save you the CSS you're applying to it 😄

Reply:
That's what I thought too but it was requiring so many overrides that I gave up. I'll take another look.

Reply:
Yeah, I don't think it's worth it. It's a lot more css because I have to adjust the header and contents separately. It's something like this (but this still needs more overrides):

```diff
diff --git a/src/amo/components/LandingPage/index.js b/src/amo/components/LandingPage/index.js
index bd641ce7..08f69c9b 100644
--- a/src/amo/components/LandingPage/index.js
+++ b/src/amo/components/LandingPage/index.js
@@ -26,6 +26,7 @@ import {
 } from 'core/utils';
 import translate from 'core/i18n/translate';
 import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
 
 import './styles.scss';
 
@@ -208,14 +209,14 @@ export class LandingPageBase extends React.Component {
 
         {errorHandler.renderErrorIfPresent()}
 
-        <div className="LandingPage-header">
-          <h1 className="LandingPage-addonType-name">
-            {headingText[addonType]}
-          </h1>
+        <Card
+          className="LandingPage-header"
+          header={headingText[addonType]}
+        >
           <p className="LandingPage-heading-content">
             {contentText[addonType]}
           </p>
-        </div>
+        </Card>
 
         <Categories addonType={addonType} />
 
diff --git a/src/amo/components/LandingPage/styles.scss b/src/amo/components/LandingPage/styles.scss
index 61c598e4..1b2f110e 100644
--- a/src/amo/components/LandingPage/styles.scss
+++ b/src/amo/components/LandingPage/styles.scss
@@ -11,18 +11,30 @@
 }
 
 .LandingPage-header {
-  background-color: $white;
-  border-radius: $border-radius-default;
+  margin: 0;
   margin-bottom: 12px;
-  padding: 24px;
-
-  @include respond-to(medium) {
-    padding: 48px 36px;
-  }
 
   @include respond-to(large) {
     margin-bottom: 24px;
   }
+
+  .Card-header {
+    margin: 0;
+    padding: 24px 24px 0;
+
+    @include respond-to(medium) {
+      padding: 48px 36px 0;
+    }
+  }
+
+  .Card-contents {
+    margin: 0;
+    padding: 0 24px 24px;
+
+    @include respond-to(medium) {
+      padding: 12px 36px 24px;
+    }
+  }
 }
 
 .LandingPage-button {
```

Reply:
Ah, damn. I just worry about that separate "card" style that won't get updated in the future.

I could see that header card component being useful in the future for pages like Language Tools as well...

But yeah, we can refactor later if needed, that seems like more overrides than original styles so your original solution is better 👍